projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -399,11 +399,11 @@ public static function filter_the_content( $content ) {
 					// First, check the image tag. Note we only check for pixel sizes now; HTML4 percentages have never been correctly
 					// supported, so we stopped pretending to support them in JP 9.1.0.
 					if ( preg_match( '#[\s"\']width=["\']?([\d%]+)["\']?#i', $images['img_tag'][ $index ], $width_string ) ) {
-						$width = false === strpos( $width_string[1], '%' ) ? $width_string[1] : false;
+						$width = ! str_contains( $width_string[1], '%' ) ? $width_string[1] : false;
```

Comment:
I wonder if we should flip these around to make it a little less confusing, like

```suggestion
						$width = str_contains( $width_string[1], '%' ) ? false : $width_string[1];
```